app/src/main/res/values/strings.xml

```diff
@@ -116,17 +116,17 @@
 
     <string name="enter_group_name">Enter group name</string>
     <string name="groups">Groups</string>
-    <string name="noGroups">Click the "+" (plus) button to add groups first.\n\nGroups make things easier to find.</string>
+    <string name="noGroups">Click the "+" (plus) button to add groups for categorization first.</string>
     <plurals name="groupCardCount">
         <item quantity="one"><xliff:g>%d</xliff:g> card</item>
         <item quantity="other"><xliff:g>%d</xliff:g> cards</item>
     </plurals>
 
     <string name="all">All</string>
-    <string name="deleteConfirmationGroup">Please confirm you want to delete this group</string>
+    <string name="deleteConfirmationGroup">Delete group?</string>
     <string name="failedOpeningFileManager">Install a file manager first.</string>
-    <string name="moveUp">Move up in list</string>
-    <string name="moveDown">Move down in list</string>
+    <string name="moveUp">Move upwards</string>
```

Comment:
Meant to ask about the arrows, sorry about that. Does this work?

Reply:
Yeah, I guess. It is still technically less correct because the list is displayed up-to-down in the group management screen, but as different tabs (thus: left-to-right) in the main screen. But I'm fine with this anyway.

Reply:
So I take it the real solution is to have separate strings for "leftwards/rightwards" in vertical mode?
Some web browsers have this problem with "close all tabs below/close all tabs to the right"

Reply:
You can't edit them on the main screen.

Let me take some screenshots to explain.

First, the manage groups activity. Here, you can create/rename/reorder/remove groups. They are displayed top to bottom:
![image](https://user-images.githubusercontent.com/1885159/113523030-a5e01280-95a5-11eb-8c37-82ca7f2dd5a7.png)

However, in the main activity, they are displayed as tabs at the top, left to right. You cannot edit them here:
![image](https://user-images.githubusercontent.com/1885159/113523034-b2646b00-95a5-11eb-80e4-95b5fb8933c0.png)